run.sh

```diff
@@ -31,18 +31,6 @@ if [ -z "$CRYOSTAT_EXT_WEB_PORT" ]; then
     CRYOSTAT_EXT_WEB_PORT="$CRYOSTAT_WEB_PORT"
 fi
 
-if [ -z "$CRYOSTAT_LISTEN_HOST" ]; then
-    CRYOSTAT_LISTEN_HOST="$CRYOSTAT_WEB_HOST"
```

Comment:
Was the `LISTEN_PORT` originally used for the two-way notifications channel?

Reply:
Sort of - it's even older than that. It was used for the original interactive "shell" execution mode, when I extended that to work over TCP sockets instead of only over stdin/stdout in a terminal.